maintainers/maintainer-list.nix

```diff
@@ -7152,6 +7152,12 @@
     githubId = 1866448;
     name = "Eric Bailey";
   };
+  Yumasi = {
+    email = "gpagnoux@gmail.com";
+    github = "Yumasi";
+    githubId = 24368641;
+    name = "Guillaume Pagnoux";
+  };
```

Comment:
You can add a public key (which I see you already have to sign your commits) in the format:

```nix
keys = [{
  longkeyid = "keytype/0xFINGERPRINT";
  fingerprint = "FINGERPRINT with spaces each 4 hexadigits";
}];

Reply:
Also, consider uploading your public key to a PGP keyserver

Reply:
Thanks for the suggestion, I added it. Actually, it is on a PGP keyserver already (pgp.mit.edu)